packages/react-components/src/components/ChatMessage/ChatMessageComponentAsEditBox.tsx

```diff
@@ -48,9 +54,18 @@ type MessageState = 'OK' | 'too short' | 'too long';
 export const ChatMessageComponentAsEditBox = (props: ChatMessageComponentAsEditBoxProps): JSX.Element => {
   const { onCancel, onSubmit, strings, message } = props;
   const [textValue, setTextValue] = useState<string>(message.content || '');
+
+  // @TODO: Remove when file-sharing feature becomes stable.
+  const getMessageAttachedFilesMetadata = (): FileMetadata[] | [] | undefined => {
+    /* @conditional-compile-remove(file-sharing) */
+    return message.attachedFilesMetadata;
+    return [];
+  };
+
+  const [attachedFilesMetadata, setAttachedFilesMetadata] = React.useState(getMessageAttachedFilesMetadata());
   const editTextFieldRef = React.useRef<ITextField>(null);
   const theme = useTheme();
-  const messageState = getMessageState(textValue);
+  const messageState = getMessageState(textValue, attachedFilesMetadata || []);
```

Comment:
nitty nit: I'd say `attachedFilesMetadata ?? []` to better reflect that the fallback is for undefined value.